ui/app/AppLayouts/Wallet/controls/CollectibleDetailsHeader.qml

```diff
@@ -8,41 +8,87 @@ import StatusQ.Core.Theme 0.1
 import StatusQ.Core 0.1
 import StatusQ.Controls 0.1
 
+import shared.controls 1.0
+
 ColumnLayout {
     id: root
-
     property alias primaryText: collectibleName.text
-    property alias secondaryText: collectibleId.text
+    property string secondaryText
+    property bool isNarrowMode
+    property string networkName
+    property string networkColor
+    property string networkIconURL
+
     property StatusAssetSettings asset: StatusAssetSettings {
-        width: 40
-        height: 40
+        readonly property int size: root.isNarrowMode ? 24 : 38
```

Comment:
we have had multiple discussion on how to handle smaller screen sizes and I think we were to check one-one basis. however making assets size smaller and bigger based on screen size doesnt seem ideal. but this needs to come from design, so again, we double check?

Reply:
Hmm interesting, this was my interpretation of the design. Do you mean what's on [Figma](https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=6455%3A122769&t=IKRFhXdnDonNMuuh-0) could be outdated?

Reply:
Hey I just saw the design and I didnt know designs for smaller screen size were already added! this is perfect then :)